src/docs/user/references/DataConnectors.adoc

```diff
@@ -318,7 +318,10 @@ Default: `HTML`
 
 The scheduler should be configured in non-fork mode to execute tasks in a single JVM rather than starting a dedicated JVM to run the task (In $PROACTIVE_HOME/config/scheduler/settings.ini, set pa.scheduler.task.fork=false).
 
-To be executed, each workflow requires the jdbc connector jar file of the used rdbms to connect to the database. Make sure you have it properly copied into the addons directory ($PROACTIVE_HOME/addons) of the scheduler.
+To be executed, each workflow requires the jdbc connector jar of the used rdbms to enable connection to the database.
```

Comment:
to enable connection to the database. => to enable the connection to the database.